Lib/test/test_traceback.py

```diff
@@ -4740,6 +4740,13 @@ def test_suggestions_extension(self):
             None
         )
 
+        self.assertRaises(
+            TypeError,
+            _suggestions._generate_suggestions,
```

Comment:
_suggestions._generate_suggestions() checks if all elements are strings before calling _Py_CalculateSuggestions(). So this test doesn't check changed code.